.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+Welcome to the Julia project!
+
+We use the GitHub issue tracker for bug reports and feature requests.
+If you have a question or are unsure if the behavior you're experiencing is a bug,
+please post to [our Discourse site](https://discourse.julialang.org).
+
+If you're submitting a bug report, be sure to include as much relevant information as
+possible, including a minimal reproducible example and the output of `versioninfo()`.
```

Comment:
Just to be clear, I was thinking about just linking the current text ("minimal reproducible example"), so no new text would be added.